app/ai/voice/agents/breeze_buddy/managers/pre_checks.py

```
@@ -303,73 +395,137 @@ async def run_pre_checks(
+
+            pre_check_exports: Dict[str, str] = {}
+            if pre_check.export_to_payload:
+                try:
+                    pre_check_exports = _extract_exports(
+                        response_data, pre_check.export_to_payload, pre_check.name
+                    )
+                    logger.info(
+                        f"Pre-check '{pre_check.name}': exported "
+                        f"{list(pre_check_exports.keys())}"
+                    )
+                except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)